spec/acceptance/nodesets/centos-6.yml

```diff
@@ -0,0 +1,10 @@
+HOSTS:
```

Comment:
Why a vagrant node if we have docker?

Reply:
If we are not testing it with TravisCI, this seems like cruft that could be removed. What do you think?

Reply:
This is done, all nodesets are docker now.